proxy/ProxySession.h

```diff
@@ -169,8 +169,8 @@ class ProxySession : public VConnection
   bool debug_on   = false;
   bool in_destroy = false;
 
-  int64_t con_id        = 0;
```

Comment:
Can we make this private?

Reply:
Why `private` instead of `protected`?

Reply:
I will make it private. 
Normally I would make this a public const variable, but we don't use constructors (yet) on this so I will resort to a private variable set in the init() and only a get accessor.